composer.json

```diff
@@ -16,21 +16,21 @@
         }
     ],
     "require": {
-        "php": ">=7.1.0",
+        "php": ">=7.1",
         "ext-simplexml": "*",
         "amphp/parallel": "~1.4",
-        "composer/composer": "1.10.6",
+        "composer/composer": "~1.10",
         "phpdocumentor/type-resolver": "~1.0 || ~1.1",
         "phpdocumentor/reflection-docblock": "~4.3 || ~5.1",
         "phpdocumentor/reflection-common": "~2.1",
         "voku/simple-cache": "~4.0",
         "nikic/php-parser": "~4.4",
         "ondrejmirtes/better-reflection": "~4.3",
-        "vimeo/psalm": "~3.11",
         "symfony/console": "~3.4 || ~4.1 || ~5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.0 || ~7.0"
+        "phpunit/phpunit": "~6.0 || ~7.0",
+        "vimeo/psalm": "~3.11"
```

Comment:
We use psalm in the code, so we can't move it into the dev section.

Reply:
Oh noooo!

`$this->typeFromPhpDocPslam = (string) @\Psalm\Type::parseString($this->typeFromPhpDoc);`

Reply:
Does it make the PHAR much bigger?

Reply:
19 MB vs. 12 MB

Reply:
Done.

Reply:
Maybe we can use https://github.com/phpstan/phpdoc-parser here, instead of the full psalm package?

Reply:
Yes, it understand some psalm stuff

https://github.com/phpstan/phpdoc-parser/search?q=psalm&unscoped_q=psalm

I am not able to execute this swap.

Reply:
I will try it, when I have time. 12 MB sounds better than 19 MB. I have to make some food for my little daughter now. :) Thanks for your testing and the fixes! :+1:

Reply:
https://github.com/voku/Simple-PHP-Code-Parser/commit/217dd6251ef2c6e63fd6eda8f6e2eefecd1b93a7 ⇉ 13 MB for the phar file :)